src/clients/mod.rs

```diff
@@ -46,6 +46,10 @@ pub fn launch_clients(config: &Config, work_receiver: Receiver<WorkItem>) -> Opt
         Protocol::Resp => {
             clients::redis::launch_tasks(&mut client_rt, config.clone(), work_receiver)
         }
+        _ => {
```

Comment:
I'd rather exhaustively list the enum variants so we have a compile time error if somebody does not add a new protocol here.